@(#)PROGRAM:SiriPrivateLearningInferencePlugin  PROJECT:SiriPrivateLearningAnalytics-1
SiriPrivateLearningInferencePlugin
LoggingConstants
Signposter
InferencePluginErrorReason
InferencePluginError
PluginRunSummary
ConversationReviewRecord
ConversationReviewTracker
CodingKeys
WindowedAsyncSequence
WindowedAsyncSequenceIterator
SessionGroundTruths
_pSg
_pSg
ScPSg
Iegn_
IeghH_
ytIeghHr_
3key_Say
G5valuet
_AASgtSg
_AASgt
GG_G
SbIegnd_
ySo8INIntentCmG
Si6offset_
7elementt
Si6offset_
7elementtSg
ySay
GSpySo8NSObjectCGSgSpyypGSgytIeglllr_
$sSY
7ElementSciQz_ABSgt
$sScI
yx_G
$sSci
13AsyncIteratorSciQz
7ElementSciQzSg
Unable to save PICS with id = %s to storage
Writing combined suggestions to disk: %s
Combining with %ld existing PLUS suggestions from disk: %s
Generated PLUS Suggestions: %s
No new ground truth generated
In SiriPrivateLearningInferencePlugin.run()
Skipping conversation with missing conversationId
Skipping conversation with existing review record, conversationId=%s, reviewRecord=%s
<Missing Conversation Id>
Failed to extract PLUS id from conversation id=%s. Emitting logs with no linking ID.
Generated Contact Prompt Ground Truth: %s
Generated Phone Call Ground Truth: %s
Classification Results: %s
Turn %ld Features: %s
Grouped %ld sessions into %ld composite sessions based on restatements.
Failed to write conversation review record for conversationId=%s. Skipping persisting ground truth.
Persisting the following ground truths to the store: %s
com.apple.siri.plus
runInferencePlugin
_TtC34SiriPrivateLearningInferencePlugin34SiriPrivateLearningInferencePlugin
privacyApprovedLoggingPeriod
privacyApprovedFollowUpDetectionPeriod
maximumTimeBetweenRestatements
maximumTimeWindowForFollowupEvents
conversationTracker
signalReader
interactionReader
groundTruthStore
picsStore
lastRunSummary
INStartCallIntent
Couldn't find session id for composite conversation. Skipping GT generation.
Generated Ground Truth for composite session %s: %s
Failed to generate a featurised session for conversationId=%s
Failed to run SiriPrivateLearningInference plugin, error=%s
[Error] Interval already ended
Unable to create directory: %s, error=%s
_TtC34SiriPrivateLearningInferencePlugin25ConversationReviewTracker
reviewedConversations
path
reviewedTimestamp
Failed to read conversation review record from file=%s, error=%s
Cannot read conversation review record from file=%s, unable to decode data
Swift/Dictionary.swift
Swift/NativeDictionary.swift
Fatal error
Duplicate values for key: '
privacyApprovedLoggingPeriod
privacyApprovedFollowUpDetectionPeriod
maximumTimeBetweenRestatements
maximumTimeWindowForFollowupEvents
conversationTracker
signalReader
interactionReader
groundTruthStore
picsStore
lastRunSummary
RawValue
unknown
failedToReadConversationData
groundTruthStoreReadError
groundTruthStoreWriteError
suggestionStoreReadError
suggestionStoreWriteError
unrecoverableError
setupError
wrappedErrorWithCause
conversationsReviewed
groundTruthGenerated
suggestionsGenerated
suggestionsUpdated
reviewedTimestamp
conversationId
reviewedConversations
path
Element
AsyncIterator
innerSequence
iterator
lastElement
finished
sessionId
groundTruth
identifier
defaultManager
createDirectoryAtURL:withIntermediateDirectories:attributes:error:
enumeratorAtPath:
initWithContentsOfURL:options:error:
InferencePlugin
